Tools/jit/_stencils.py

```diff
@@ -348,9 +325,21 @@ def _emit_global_offset_table(self) -> None:
             )
             self.data.body.extend([0] * 8)
 
+    def _get_trampoline_mask(self) -> str:
+        bitmask: int = 0
+        trampoline_mask: list[str] = []
+        for ordinal in self.trampolines:
+            bitmask |= 1 << ordinal
+        if bitmask:
+            trampoline_mask = [
+                f"0x{(bitmask >> i*32) & ((1 << 32) - 1):x}"
+                for i in range(0, SYMBOL_MASK_SIZE)
+            ]
+        return ", ".join(trampoline_mask)
+
     def as_c(self, opname: str) -> str:
         """Dump this hole as a StencilGroup initializer."""
-        return f"{{emit_{opname}, {len(self.code.body)}, {len(self.data.body)}}}"
+        return f"{{emit_{opname}, {len(self.code.body)}, {len(self.data.body)}, {{{self._get_trampoline_mask()}}}}}"
```

Comment:
This is a lot of braces, haha. Maybe move the braces to `_get_trampoline_mask`?

```suggestion
        return f"{{emit_{opname}, {len(self.code.body)}, {len(self.data.body)}, {self._get_trampoline_mask()}}}"
```

...and then turn `", ".join(trampoline_mask)` to `"{" + ", ".join(trampoline_mask) + "}"` above. That makes it a bit clearer that it's returning valid C code.

Reply:
Done.